editor/questionbankcontroller.js

```diff
@@ -472,6 +472,57 @@ angular.module('createquestionapp', [])
         ecEditor.dispatchEvent($scope.pluginIdObj.question_create_id + ":showpopup", questionObj);
       }
     }
+
+    $scope.deleteQuestion = function(questionObj){
+      $scope.assessmentId = questionObj.IL_UNIQUE_ID;
```

Comment:
I don't remember using this reference before `questionObj.IL_UNIQUE_ID`. Is this the same as `questionObj.identifier`?